pyriemann/utils/test.py

```diff
@@ -55,6 +55,36 @@ def is_skew_sym(X):
     return is_square(X) and np.allclose(X, -np.swapaxes(X, -2, -1))
 
 
+def is_hankel(X):
```

Comment:
can you add tiny tests for this function? I know it's used elsewhere but my rule of thumb is that any public function in foo.py should be tested in tests/test_foo.py